docs/debug.rst

```diff
@@ -4,65 +4,20 @@
 Debugging
 =========
 
-Debugging webapp
-================
-
-There are some kinds of problems that are hard to debug because gunicorn
-won't start up and doesn't tell you anything.
-
-
-Run without gunicorn
---------------------
-
-Try running the webapp without gunicorn:
-
-.. code-block:: shell
-
-   $ docker-compose run --rm --service-ports web shell
-   app@78dd072d4a88:/app$ python ichnaea/webapp/app.py
-   running at http://localhost:8000
-   ...
-
-
-If there is a bug in the webapp that's preventing the webapp from starting up,
-it'll spit out a traceback.
-
-
-Run gunicorn with preload
--------------------------
-
-You can also run gunicorn with ``--preload`` set. Edit ``docker/run_web.sh`` to
-pass ``-preload``. You'll also need to change the worker class to ``sync``
-because preload doesn't work with gevent. Sometimes that'll help debugging
-webapp issues.
-
-
-Switch gunicorn logging level to debug
---------------------------------------
-
-You can tell gunicorn to log at debug level by adding this to your ``my.env``::
-
-    GUNICORN_LOGLEVEL=debug
-
-Then run ``make run``.
-
-
-.. Note:: 2019-08-16: Below this needs to be updated.
-
-
 MySQL Config
 ------------
 
-First, lets check the database configuration. If you have a development
-setup, you can open a MySQL client prompt via:
+First, lets check the database configuration.
+
+In a local development environment, you can run the mysql client like this:
 
 .. code-block:: bash
 
-    docker-compose exec mysql mysql -uroot -plocation location
+   make mysql
```

Comment:
Nice, didn't know this one 😀

Reply:
It might be new-ish. I forget when I added it. I'm wondering if we should add one for redis-cli, too.